lib/openid/cryptutil.rb

```diff
@@ -37,23 +25,15 @@ def CryptUtil.sha1(text)
     end
 
     def CryptUtil.hmac_sha1(key, text)
-      if Digest.const_defined? :HMAC      
-        Digest::HMAC.new(key,Digest::SHA1).update(text).digest
-      else
-        return HMAC::SHA1.digest(key, text)
-      end
+      OpenSSL::HMAC.hexdigest('sha1', key, text)
```

Comment:
This breaks the fallback on SHA! but I won't insist on re-introducing it since it'll be a short-live solution.